static/settings.ts

```diff
@@ -205,7 +205,6 @@ export class Settings {
                 private subLangId: string | null) {
 
         this.settings = settings;
-        this.settings.defaultLanguage = this.settings.defaultLanguage === null ? undefined : settings.defaultLanguage;
```

Comment:
This probably comes from old behaviour creeping in to the new code. I'll double-check tomorrow if this is still needed

Reply:
I've checked, this looks ok